buildkite/bazelci.py

```diff
@@ -765,17 +765,16 @@ def execute_commands(
         if not build_targets and not test_targets:
             raise BuildkiteException("There are neither build nor test targets")
 
+        include_json_profile = task_config.get("include_json_profile", [])
```

Comment:
I would also be fine with just a boolean option, all or nothing.
But I leave that decision to you and the engprod team.

Reply:
I thought of that too but opted for the flexibility instead.